cds_sorenson/api.py

```diff
@@ -97,7 +105,7 @@ def get_encoding_status(job_id):
     :param job_id: string with the job ID.
     :returns: tuple with the status message and progress in %.
     """
-    SORENSON_STATUSES = {
+    statuses = {
```

Comment:
This doesn't conform to the pep8 naming conventions. Should I move it to the module level?